test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommandTests.cs

```diff
@@ -1913,8 +1913,7 @@ public async Task RestoreCommand_RestoreExactVersionWithFailingSourceAsync()
             // Arrange
             var sources = new List<PackageSource>
             {
-                new PackageSource("https://failingSource"),
-                new PackageSource(NuGetConstants.V3FeedUrl)
```

Comment:
Why did you remove this?

Reply:
let me go debug again. used to fail fast...and raise exception. 
when i changed it to be a proper error, at the time of the assert the exception hadn't happened yet.

Reply:
debugging again...i changed the test back and am adding checks for NU1300, etc... in the assert section.
i had some different exception handling routines sometime between friday and monday...that i enventually got rid of.
THat is likely why i had the problem. i shouldn't have changed the packagesources. thx for asking.

Reply:
fixed in next iteration.